### ISSUE TITLE
0.9.1: Unhandled Exception: PlatformException(videoRendererDispose, videoRendererDispose(): render [1] not found !, null, null)

### ISSUE THREAD

**Issue by AmirhosseinShk** [flutter-webrtc/flutter-webrtc#1036]
when dispose videoRenderer on Android 12 get error that tell render [1] not found!

error happened when u want dispose videoRenderer after initialize it.

E/flutter ( 9330): [ERROR:flutter/lib/ui/ui_dart_state.cc(198)] Unhandled Exception: PlatformException(videoRendererDispose, videoRendererDispose(): render [1] not found !, null, null)
E/flutter ( 9330): #0      StandardMethodCodec.decodeEnvelope (package:flutter/src/services/message_codecs.dart:607:7)
E/flutter ( 9330): #1      MethodChannel._invokeMethod (package:flutter/src/services/platform_channel.dart:167:18)
E/flutter ( 9330): <asynchronous suspension>
E/flutter ( 9330): #2      RTCVideoRenderer.dispose (package:flutter_webrtc/src/native/rtc_video_renderer_impl.dart:61:5)
E/flutter ( 9330): <asynchronous suspension>
E/flutter ( 9330): #3      CallRepo.disposeRenderer (package:deliver/repository/callRepo.dart:1433:7)
E/flutter ( 9330): <asynchronous suspension>

D:\flutter\bin\flutter.bat doctor --verbose
[√] Flutter (Channel stable, 3.0.5, on Microsoft Windows [Version 10.0.19044.1826], locale en-US)
    • Flutter version 3.0.5 at D:\Work\flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision f1875d570e (3 weeks ago), 2022-07-13 11:24:16 -0700
    • Engine revision e85ea0e79c
    • Dart version 2.17.6
    • DevTools version 2.12.2

[√] Android toolchain - develop for Android devices (Android SDK version 33.0.0)
    • Android SDK at C:\Users\AmirShk\AppData\Local\Android\sdk
    • Platform android-33, build-tools 33.0.0
    • Java binary at: C:\Program Files\Android\Android Studio\jre\bin\java
    • Java version OpenJDK Runtime Environment (build 11.0.8+10-b944.6842174)
    • All Android licenses accepted.

[√] Chrome - develop for the web
    • Chrome at C:\Program Files (x86)\Google\Chrome\Application\chrome.exe

[√] Visual Studio - develop for Windows (Visual Studio Enterprise 2019 16.11.8)
    • Visual Studio at C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise
    • Visual Studio Enterprise 2019 version 16.11.32002.261
    • Windows 10 SDK version 10.0.19041.0

[√] Android Studio (version 4.2)
    • Android Studio at C:\Program Files\Android\Android Studio
    • Flutter plugin can be installed from:
       https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
       https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.8+10-b944.6842174)

[√] IntelliJ IDEA Ultimate Edition (version 2022.1)
    • IntelliJ at C:\Program Files\JetBrains\IntelliJ IDEA 2022.1.2
    • Flutter plugin version 68.1.4
    • Dart plugin version 221.5787.37

[√] VS Code, 64-bit edition (version 1.69.0)
    • VS Code at C:\Program Files\Microsoft VS Code
    • Flutter extension can be installed from:
       https://marketplace.visualstudio.com/items?itemName=Dart-Code.flutter

[√] Connected device (4 available)
    • SM A515F (mobile) • 192.168.1.7:5555 • android-arm64  • Android 12 (API 31)
    • Windows (desktop) • windows          • windows-x64    • Microsoft Windows [Version 10.0.19044.1826]
    • Chrome (web)      • chrome           • web-javascript • Google Chrome 102.0.5005.115
    • Edge (web)        • edge             • web-javascript • Microsoft Edge 103.0.1264.71

[√] HTTP Host Availability
    • All required HTTP hosts are available

• No issues found!

* **OS**: Android
* **OS version**: 12

**Comment by AmirhosseinShk** [comment]
my problem because VideoRenderer Object initialize 2 times and when i want dispose it get error.

**Comment by leeyisoft** [comment]
> 阿米尔侯赛因史克

I also encountered the same mistake, could you tell me how you dealt with it?

**Comment by cloudwebrtc** [comment]
fixed https://github.com/flutter-webrtc/flutter-webrtc/commit/939367f4c74517fdf68c29f70e24c9e32f849ea6

**Comment by leeyisoft** [comment]
> fixed [939367f](https://github.com/flutter-webrtc/flutter-webrtc/commit/939367f4c74517fdf68c29f70e24c9e32f849ea6)

Thank you very much. I mainly got this update when I came to work this morning. I will try it at home this evening

**Comment by leeyisoft** [comment]
There is an error: 
```

[   +3 ms] D/FlutterWebRTCPlugin(21098): mediaStreamTrackStop() track is null
[        ] D/FlutterWebRTCPlugin(21098): mediaStreamTrackStop() track is null
[        ] E/flutter (21098): [ERROR:flutter/lib/ui/ui_dart_state.cc(198)] Unhandled Exception: PlatformException(videoRendererSetSrcObject, videoRendererSetSrcObject(): render [7] not found !, null, null)
[        ] E/flutter (21098): #0      StandardMethodCodec.decodeEnvelope (package:flutter/src/services/message_codecs.dart:607:7)
[        ] E/flutter (21098): #1      MethodChannel._invokeMethod (package:flutter/src/services/platform_channel.dart:167:18)
[        ] E/flutter (21098): <asynchronous suspension>
[        ] E/flutter (21098): 
[        ] D/FlutterWebRTCPlugin(21098): mediaStreamRelease() mediaStream is null
```

<img width="1194" alt="image" src="https://user-images.githubusercontent.com/6138668/190660486-140df06e-c4f3-4113-87a2-31f81d3d8b63.png">
